model/src/main/scala/hmda/model/census/Census.scala

```diff
@@ -0,0 +1,65 @@
+package hmda.model.census
+
+object Census {
+  val states = Map(
```

Comment:
Yes there is. We will need to provide data structures from Census for validation. I'm not sure what it will look like in the end, so this object might change. One possibility is to have a `State`, `County`, etc. object that encapsulates the necessary fields. This is probably going to be refactored at some point